.github/workflows/check-api-go.yml

```diff
@@ -0,0 +1,68 @@
+name: Check api-go compatibility
+
+on:
+  workflow_dispatch:
+    inputs:
+      sdk_commit:
+        description: sdk-go commit to check
+        required: true
+        default: master
```

Comment:
This needs to be the latest tag. The goal is to make sure the latest tag of SDK Go works with the latest (coming) tag of API Go.